setup.py

```diff
@@ -1,71 +1,38 @@
-from setuptools import setup, findall
```

Comment:
Probably best to port over the setup.py from the develop branch, and apply the changes you have here.